parlai/scripts/self_chat.py

```diff
@@ -30,92 +28,97 @@ def setup_args(parser=None):
     )
     parser.add_argument(
         '-it',
-        '--interactive-task',
+        '--selfchat-task',
         type='bool',
         default=True,
-        help='Create interactive version of task',
+        help='Create a self chat version of the task',
+    )
+    parser.add_argument(
+        '--num-self-chats', type=int, default=1, help='Number of self chats to run'
     )
     parser.add_argument(
         '--selfchat-max-turns',
         type=int,
-        default=10,
-        help="The number of dialogue turns before self chat ends.",
+        default=6,
+        help='The number of dialogue turns before self chat ends',
     )
     parser.add_argument(
         '--seed-messages-from-task',
         action='store_true',
-        help="Automatically seed conversation with messages from task dataset.",
+        help='Automatically seed conversation with messages from task dataset.',
+    )
+    parser.add_argument(
+        '--outfile', type=str, default=None, help='File to save self chat logs'
     )
-    parser.add_argument('--outfile', type=str, default='/tmp/selfchat.json')
     parser.add_argument(
-        '--format', type=str, default='json', choices={'parlai', 'json'}
+        '--save-format',
+        type=str,
+        default='conversations',
+        choices=['conversations', 'parlai', 'json'],
```

Comment:
this is actually a bug that i found in my experience with self-chat (it's in one of my other PRs) but i think you need to change `json` to `jsonl` here